src/empty-state-card.tsx

```diff
@@ -70,24 +70,32 @@ const EmptyStateCard: React.FC<Props> = ({
         image = <img className={classes.image} alt="" src={imageUrl} />;
     }
 
+    let resizedIcon;
+    if (icon) {
+        resizedIcon = React.cloneElement(icon, {size: isTabletOrSmaller ? 64 : 80, color: colors.brand});
+    }
+
+    let smallButton;
+    if (button) {
+        smallButton = React.cloneElement(button, {small: true} as Partial<React.FC<ButtonProps>>);
```

Comment:
I prefer to not modify the received component, instead of this we could make a runtime check and show an error if not a production build, so it will show that error while developing and make tests fail when using a wrong combination